src/remoteConfig.ts

```diff
@@ -22,6 +22,7 @@ type RemoteConfigValue = {
   enableBalance: boolean;
   approvedStoreKitSandboxUsers: string[];
   coreProductId: string;
+  coreModifier: number;
```

Comment:
let's just add if coreModifier is undefined we don't run `assignSubscriptionCores`